README.md

```diff
@@ -125,8 +125,8 @@ Then run `bundle install`.
 
 Now you need to configure the gem by telling it:
 
-* which matchers you want to use in your tests
-* that you're using Minitest so that it can make those matchers available in
+- which matchers you want to use in your tests
```

Comment:
Why are we replacing * with _? I don't mind, just curious.

Reply:
This was my linter, my mistake. Lemme revert this heh